samples/matter/template/README.rst

```diff
@@ -32,7 +32,7 @@ IPv6 network support
 
 The development kits for this sample offer the following IPv6 network support for Matter:
 
-* Matter over Thread is supported for the ``nrf52840dk/nrf52840``, ``nrf5340dk/nrf5340/cpuapp``, ``nrf21540dk/nrf52840``, ``nrf54l15dk/nrf54l15/cpuapp``, and ``nrf54l15dk/nrf54l10/cpuapp`` board targets.
+* Matter over Thread is supported for the ``nrf52840dk/nrf52840``, ``nrf5340dk/nrf5340/cpuapp``, ``nrf21540dk/nrf52840``, ``nrf54l15dk/nrf54l15/cpuapp``, ``nrf54l15dk/nrf54l10/cpuapp``, and ``nrf54lm20pdk/nrf54lm20a/cpuapp`` board targets.
```

Comment:
```suggestion
* Matter over Thread is supported for the following board targets:

  * ``nrf52840dk/nrf52840``
  * ``nrf5340dk/nrf5340/cpuapp``
  * ``nrf21540dk/nrf52840``
  * ``nrf54l15dk/nrf54l15/cpuapp``
  * ``nrf54l15dk/nrf54l10/cpuapp``
  * ``nrf54lm20pdk/nrf54lm20a/cpuapp``
```
This might improve the readability a bit.